.changeset/dull-frogs-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/oauth-providers': minor
```

Comment:
Sorry! I could not tell you correctly. It's a patch change. So rewrite `minor` to `patch`.

Reply:
@yusukebe Fixed again 👍🏽